### DISCUSSION TITLE
Github education requiring wrong school email

### DISCUSSION THREAD

**Discussion by Shin1ma** [community/community#143565] [GitHub Education]
### Hi everyone,

Hi! Finally got a hold of the documents required to access github education and was trying to apply. My school is the IIS "Giorgi - Woolf" institute of high education and has the official e-mail at ".giorgiwoolf.edu.it", I selected it from the suggested schools, it has the same website url too, but the required e-mail to apply is ".giorgimi.edu.it", I've never heard of that e-mail being used in the school, even in past years. 
![Screenshot 2024-11-04 194700](https://github.com/user-attachments/assets/39d4e340-59ac-4125-a0e2-3cbe3bcda52e)

**Comment by davevad93** [comment]
Hi @Shin1ma ,

It's possible that GitHub has incorrect or old information about your school, I suggest you to contact support by opening a ticket [here](https://support.github.com/contact/education), select "`My selected school has incorrect or incomplete information e.g. domains or campus location`" and explain your issue.

**Reply by Shin1ma** [reply]
Thanks! I'll do so.
